python/lsst/pipe/tasks/makeCoaddTempExp.py

```diff
@@ -117,13 +117,19 @@ def run(self, patchRef, selectDataList=[]):
             except (KeyError, ValueError):
                 visitId = i
 
-            exp = self.createTempExp(calexpRefList, skyInfo, visitId)
-            if exp is not None:
+            exps = self.createTempExp(calexpRefList, skyInfo, visitId)
```

Comment:
Oh you probably meant a Struct of a dict.  OK, that would work here.  With the future assembleCoadd I don't want to give the impression that direct, psf-matched and likelihood warps are interchangeable.